sdk/servicebus/azure_messaging_servicebus/src/error.rs

```diff
@@ -53,16 +53,16 @@ impl fmt::Display for ErrorKind {
 }
 
 /// A Service Bus specific error.
-#[derive(SafeDebug, Clone, PartialEq, Eq)]
```

Comment:
The error type should just be `Debug` not `SafeDebug`. All the information is kinda important and you should make sure nothing inside it otherwise leaks. `typespec::Error` impl `Debug`.